pages/docs/environments/local-development.mdx

```diff
@@ -79,7 +79,7 @@ docker compose down
 
 ### Troubleshooting
 
-If you are getting an error, try to rebuild the images with `docker compose up --build`. If that doesn't work, stop the containers with `docker compose down` and remove the stopped BioDrop container with `docker rm BioDrop`. Then, remove the images with `docker rmi BioDrop-app`. Lastly, rebuild the images with `docker compose up --build`.
+If you are getting an error, try to rebuild the images with `docker compose up --build`. If that doesn't work, stop the containers with `docker compose down` and remove the stopped BioDrop container with `docker rm BioDrop`. Then, remove the images with `docker rmi biodrop-app`. Lastly, rebuild the images with `docker compose up --build`.
```

Comment:
Should this also be changed?

```suggestion
If you are getting an error, try to rebuild the images with `docker compose up --build`. If that doesn't work, stop the containers with `docker compose down` and remove the stopped BioDrop container with `docker rm biodrop-app`. Then, remove the images with `docker rmi biodrop-app`. Lastly, rebuild the images with `docker compose up --build`.
```

Reply:
Hey,
No, it should be `biodrop` as it's a container name and `biodrop-app` is an image name. Yes, we need to fix the capitalization from `BioDrop` to `biodrop` cc @Shiva-Sai-ssb.

Reply:
Hey,

No, it should be `biodrop` as it's a container name and `biodrop-app` is an image name. Yes we need to fix the capitalization from `BioDrop` to `biodrop` cc @Shiva-Sai-ssb

Reply:
Thank you Pradumna for clarifying  👍